.github/workflows/release.yml

```diff
@@ -221,7 +221,7 @@ jobs:
 
           "" | python configure.py
 
-          bazelisk --output_base=C:\build_output build :build_pip_pkg --enable_runfiles --discard_analysis_cache --notrack_incremental_state --nokeep_state_after_build --nouse_action_cache --local_ram_resources=4096
+          bazelisk --output_base=C:\build_output build :build_pip_pkg --compilation_mode=fastbuild --enable_runfiles --discard_analysis_cache --notrack_incremental_state --nokeep_state_after_build --nouse_action_cache --local_ram_resources=4096
```

Comment:
Do you think it is worth trying to remove the `--nouse_action_cache` flag? Not sure if it also slows down things for non-incremental builds here.

Reply:
Unfortunately this suggestion doesn't help, it also doesn't make it through in time.